chapters/en/module5.md

```diff
@@ -40,23 +37,23 @@ Right, but there is also another correct option.
 <opt text='<code>origin</code>'>
 <code> origin </code> is referring to the remote GitHub repository.
 </opt>
-<opt text='2, 3 and 4 are correct'>
+<opt text='<code>master</code>'>
+Right, but there is also another correct option.
+</opt>
+<opt text='The three first options are all correct'>
 <code>main</code>, <code>origin</code> and <code>HEAD</code> are labels that provide valuable information to the user when checking the project history, but two of them not necessary refer to the default branch.
 </opt>
-<opt text='1 and 3 are correct' correct='true'>
+<opt text='The second and fourth options are both correct' correct='true'>
```

Comment:
Hmm I think this should be "second and fourth" now because I moved the "master" option from first to last:
![image](https://user-images.githubusercontent.com/4560057/183576035-1804eaaa-ef0c-4bf9-b090-5296daa8b205.png)

Reply:
yes, I noticed it later. Thank you